db/schema.rb

```diff
@@ -24,4 +42,30 @@
     t.datetime "updated_at"
   end
 
+  create_table "links", force: :cascade do |t|
+    t.string  "content_id"
+    t.integer "version",    null: false
+    t.json    "links"
```

Comment:
`link.links` is a confusing structure, especially since a "link" is actually a representation of multiple links.